.gitignore

```diff
@@ -4,3 +4,11 @@
 
 *egg-info
 ._*
+.idea
```

Comment:
please put developer specific ignores into your private gitignore file
e.g. in your ~/.gitconfig write sth like
```
[core]
        excludesfile = /home/username/.gitignore
```
and in ~/.gitignore write your own rules